server/testutils/containere2e/containere2e.go

```diff
@@ -2,29 +2,38 @@ package containere2e
 
 import (
 	"context"
-	"encoding/json"
-	"net/http"
 	"os"
 	"os/exec"
 	"path/filepath"
+	"strings"
 	"sync"
 	"testing"
 
 	"github.com/mattermost/mattermost-plugin-msteams/server/store/sqlstore"
 	"github.com/mattermost/mattermost-plugin-msteams/server/testutils/mmcontainer"
-	"github.com/mattermost/mattermost-plugin-msteams/server/testutils/testmodels"
-	"github.com/mattermost/mattermost/server/public/model"
 	"github.com/stretchr/testify/require"
+	"github.com/testcontainers/testcontainers-go"
+	"github.com/testcontainers/testcontainers-go/modules/mockserver"
+	"github.com/testcontainers/testcontainers-go/network"
 )
 
+type tLogConsumer struct {
+	t *testing.T
+}
+
+func (tlc *tLogConsumer) Accept(log testcontainers.Log) {
+	tlc.t.Log(strings.TrimSpace(string(log.Content)))
+}
+
 var buildPluginOnce sync.Once
 
 func buildPlugin(t *testing.T) {
 	cmd := exec.Command("make", "-C", "../../", "dist")
 	cmd.Env = os.Environ()
 	cmd.Env = append(cmd.Env, "DEFAULT_GOOS=linux")
 	cmd.Env = append(cmd.Env, "DEFAULT_GOARCH=amd64")
-	cmd.Env = append(cmd.Env, "GO_BUILD_TAGS=clientMock")
+	cmd.Env = append(cmd.Env, "MM_SERVICESETTINGS_ENABLEDEVELOPER=true")
```

Comment:
Note that this implies different semantics at runtime, i.e. if we had license checks they would "change" with this setting. Is this strictly necessary?

Reply:
No, I was trying to speed up the compilation time, but I can remove it. For sure.